### ISSUE TITLE
[bug] permisions getting applied only in menu.

### ISSUE THREAD

**Issue by dionys1705** [bolt/bolt#6694]
Hello,

Ive found a bug in the permisions settings for content type's
when I set the permisions for 1 specific content type it does apply the new permisions. However in the backend it does apply it in the side menu. And when I am in the overview page it is not applied. screenshots below.

Details
-------

 - Bolt Version: [ 3.2.6 ]
 - Install type: [ GitHub checkout ]
 - PHP version: [ 7.1.5 ]
 - Used web server: [ Apache 2.4 ]
 - For UX/UI issues: [ google chrome. version 58.0.3029.110 (64-bit) ]

Reproduction
------------
When I change the rights of an content type, I can acces it with updated rights trough option 1. but not option 2. ( option 1 and 2 will be screenshots bellow. )

URL: https://example.org/bolt/overview/verhalen ( <-- verhalen is display name the actual content type is "stories" )

here are the applied rights, this is good. I can create stories and change them by clicking the name and after changes safe them.
![image](https://cloud.githubusercontent.com/assets/28395146/26248137/5d1c3ad0-3ca1-11e7-9087-5508049ae0aa.png)
However, here in the overvieuw the rights changes havent been applied. The "new" button is not here.
![image](https://cloud.githubusercontent.com/assets/28395146/26248205/9cfd8f3c-3ca1-11e7-8966-83f26c294d70.png)
also for the above^ screenshot. If I click edit it gives me the error message "You do not have sufficient rights to do this.":
![image](https://cloud.githubusercontent.com/assets/28395146/26248259/e8333024-3ca1-11e7-926e-c11cf86132fd.png)

This is the content type: 
```
stories:
    name: Verhalen
    singular_name: Verhaal
    singular_slug: verhalen
    slug: verhalen
    fields:
        title:
            type: text
            group: content
            label: "Titel"
        slug:
            type: slug
            uses: title
            group: content
            label: "URL pagina"
            postfix: "Niet handmatig invullen dit gaat vanzelf"
        name:
            type: text
            group: content
            label: "Naam"
        body:
            type: html
            group: content
            label: "Verhaal"
        images:
            type: imagelist
            group: content
            label: "Foto's"
        quote:
            type: text
            group: content
            label: "Quote"
            postfix: "Dit word in een Quote gedeelte weergegeven. "
        meta_title:
            type: text
            group: SEO
            label: "Meta titel"
        meta_keywords:
            type: text
            group: SEO
            label: "Meta keywords"
        meta_description:
            type: textarea
            height: 50px
            group: SEO
            label: "Meta description"
    default_status: published
    listing_template: stories.twig
    record_template: story.twig
    listing_records: 9999
```

And these are the rights:
  ```
  stories:
        edit: [ developer, admin, chief-editor ]
        create: [ developer, admin, chief-editor ]
        publish: [ developer, admin, chief-editor ]
        depublish: [ developer, admin, chief-editor ]
        delete: [ developer, admin ]
        change-ownership: [ developer ]
        view: [ developer, admin, chief-editor ]
​
```
 **Steps to reproduce**:
1. Create a content type.
2. Set the rights for this content type.
3. Create an account that does have sufficient rights to create/edit the content type.
4. Try to edit the content type.
Done. this got me the [bug].

**Expected result**: 
   Thought I would be able to create a new story from the overview page or go to edit mode from the 
   overvieuw page.

**Actual result**: 
   I can not enter the edit mode or create a new story form the overview page.

Account rights:
![image](https://cloud.githubusercontent.com/assets/28395146/26248810/2015dbd4-3ca4-11e7-998a-7ebb8d1c5b58.png)

**Comment by bobdenotter** [comment]
Hi @dionys1705, 

Could you try updating to Bolt 3.2.13? I've tried to reproduce this, but wasn't succesful.

I've copied the contenttype and your permissions, added a new "Chief editor", and logged in with that user. I get this, on the overview screen:

![screen shot 2017-05-21 at 10 29 12](https://cloud.githubusercontent.com/assets/1833361/26282343/7b8e5974-3e10-11e7-9606-5bca8d485697.png)

I can click the "new" button, and create a new record just fine.

**Comment by dionys1705** [comment]
hi, @bobdenotter 

Updating to 3.2.13 did not work for me. The +New button still isnt here and I cant edit the stories.

![image](https://cloud.githubusercontent.com/assets/28395146/26297605/dce5ce46-3ed3-11e7-8a35-2d86b3bc105e.png)
![image](https://cloud.githubusercontent.com/assets/28395146/26297619/e67df366-3ed3-11e7-8d01-a89f6de2c94b.png)


So is there a way to completely reset the rights settings? it seems that if I change the permisions back. Or put in the standard permisions file it just keeps applying the settings I had set in the persmissions.yml file. 

- Ive deleted the permisions file completely and added the basic permissions.yml file.
- Cleared cache.

**Comment by dionys1705** [comment]
Hello, @bobdenotter 

Ive fixed the problem.
I went to the SQL database. and just went to delete everything there ( There was no importand content in there. ) after that I just checked database again, created the tables and cleared cache. after all this **Without changing a thing in the content types, permissions or the backend**, it worked completelly fine. 

I do not know for sure if this is right but I think the "Check database" did something wrong the first time it created the tables. 

Hope this is helpfull to other people.
